experiments/compositions/samples/otherCloud/composition/crdBuilder/config/crd/bases/zicong.my.comapny.com_attachedakses.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: attachedakses.zicong.my.comapny.com
+spec:
+  group: zicong.my.comapny.com
+  names:
+    kind: AttachedAKS
+    listKind: AttachedAKSList
+    plural: attachedakses
+    singular: attachedaks
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: AttachedAKS is the Schema for the attachedaks API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: AttachedAKSSpec defines the desired state of AttachedAKS
+            properties:
+              adminUsers:
+                items:
+                  type: string
+                type: array
+              attachedPlatformVersion:
+                type: string
+              azureRegion:
+                type: string
+              gcpProjectNumber:
+                type: string
+              gcpRegion:
+                type: string
+              kubernetesVersion:
+                type: string
+            type: object
```

Comment:
Please Create a Facade API to create CRDs., We don't need the whole crdBuilder folder. 

 refer: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/experiments/compositions/samples/CloudSQL/composition/hasql.yaml#L16

Reply:
Please note that the `inputAPIGroup` in compositions change to something like `  inputAPIGroup: cloudsqls.facade.compositions.google.com
`